scripts/dev/deploy-operator.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
This script is a pure 'dev' thing, correct? Doesn't seem to be used by e2e

Reply:
Yeah, that's for my laziness. And that's the reason why its in `dev` folder :)

Reply:
Ah! didn't notice the folder :)